docs/links.md

```diff
@@ -7,6 +7,6 @@ This file contains links to all the HNC documents (sorted by last updated).
 - [2019/12] [HNC CLI naming](http://bit.ly/hnc-cli-naming)
 - [2020/01] [HNC self-service namespaces UX](http://bit.ly/hnc-self-serve-ux)
 - [2020/01] [HNC type configuration](http://bit.ly/hnc-type-configuration)
-- [2020/02] [Metrics user guide: Stackdriver on GKE](https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/hnc/doc/metrics/stackdriver-gke.md)
+- [2020/02] [Metrics user guide: Stackdriver on GKE](https://github.com/kubernetes-sigs/hierarchical-namespaces/blob/master/doc/metrics/stackdriver-gke.md)
```

Comment:
Could probably just change this to `metrics/stackdriver-gke.md` since we're already in the `docs/` directory.

Reply:
The link is a 404 in the old repo as well, created https://github.com/kubernetes-sigs/hierarchical-namespaces/issues/7 to investigate and fix the link.